config.ru

```diff
@@ -1,12 +1,28 @@
 require 'rubygems'
 require 'bundler/setup'
+require 'routemaster/client'
 require 'routemaster/receiver'
 require 'dotenv'
 require 'sinatra'
 require 'pry'
 
 Dotenv.load!
 
+subscriptions = ENV.fetch('TOPIC_SUBSCRIPTIONS', '').split(',')
+if subscriptions.any?
+  routemaster = Routemaster::Client.new(
+    url: ENV.fetch('ROUTEMASTER_URL'),
+    uuid: ENV.fetch('ROUTEMASTER_UUID', 'demo'),
+  )
+
+  routemaster.subscribe(
+    topics: subscriptions,
+    callback: ENV.fetch('CALLBACK_URL'),
+    uuid: ENV.fetch('CLIENT_UUID', 'demo'),
+    max: 1
+  )
```

Comment:
Also, why would it fail if the topic doesn't exist. Is it not possible to subscribe to a routemaster topic before the first event has been sent. (AFAIK topics are created when the first event is sent?)